src/components/step-nav.scss

```diff
@@ -0,0 +1,197 @@
+@import "../base/config.scss";
+
+//------------------------------------------------------------//
+
+// * StepNav Component
+// * 1. Settings
+// * 2. Define Default StepNav
+// * 3. Small Modifier
+// * 4. Title Modifier
+// * 5. Responsive Mixin
+
+// TODO : document global dependencies
+// TODO : Add Small StepNav Modifier
+// TODO : Add Titled Modifier
+
+//------------------------------------------------------------//
+
+// * 1. Settings
+
+$StepNav-size: 60px;
+$StepNav-border: 4px;
+
+$StepNav-size--small: 40px;
+$StepNav-border--small: 2px;
+
+// * 2. Define Default StepNav
+
+.StepNav {
+  padding-left: ($StepNav-size * 1.25);
+  position: relative;
+  @include clear();
+  padding-bottom: $vertical-block-spacing--small;
+  &.steps-2 {
```

Comment:
These `steps-*` feel like `.StepNav` modifiers to me. The numbers are a little tricky with naming since they don't have a case, but these could be something like `StepNav--4Steps` or `.StepNav--fourSteps`... and wouldn't need to be nested within the `StepNav` because the naming convention connects them as a modifier.